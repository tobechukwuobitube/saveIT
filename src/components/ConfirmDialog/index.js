import React from 'react';

import '../../assets/css/Dialog.css';
import '../../assets/css/LoginForm.css';


const ConfirmDialog = ({ hideConfirmDialog, createAccount, account, loading }) => (
  <div id='dlg-wrapper'>
    <div id='dlg-box'>
      <div id='dlg-header'>
        Creating Account
        {loading ? 
          <div style={{ float: 'right' }} className='loader' /> 
          : 
          <div />}
      </div>
      <div id='dlg-body'>
        You are creating an account:
        <span>{' '}
          Account <span id='accountType'>{account}</span>
        </span>
      </div>
      <div id='dlg-footer'>
        
        <button className='btn_n' onClick={hideConfirmDialog}>
          Cancel
        </button>
        <button className='btn_n' onClick={createAccount}>
          Proceed
        </button>
      </div>
    </div>
  </div>
);

export default ConfirmDialog;
