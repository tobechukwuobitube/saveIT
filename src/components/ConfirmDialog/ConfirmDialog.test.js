import React from 'react';
import renderer from 'react-test-renderer';
import ConfirmDialog from './index';

test('ConfirmDialog snapshot test', () => {
  const wrapper = renderer.create(<ConfirmDialog />);
  expect(wrapper).toMatchSnapshot();
});
