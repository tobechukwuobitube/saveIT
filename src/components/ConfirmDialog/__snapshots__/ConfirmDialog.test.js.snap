// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ConfirmDialog snapshot test 1`] = `
<div
  id="dlg-wrapper"
>
  <div
    id="dlg-box"
  >
    <div
      id="dlg-header"
    >
      Creating Account
      <div />
    </div>
    <div
      id="dlg-body"
    >
      You are creating an account:
      <span>
         
        Account 
        <span
          id="accountType"
        />
      </span>
    </div>
    <div
      id="dlg-footer"
    >
      <button
        className="btn_n"
      >
        Cancel
      </button>
      <button
        className="btn_n"
      >
        Proceed
      </button>
    </div>
  </div>
</div>
`;
